zerver/tests/test_auth_backends.py

```diff
@@ -1887,6 +1887,7 @@ def test_social_auth_invited_as_admin_but_expired(self) -> None:
             iago,
             [email],
             [],
+            send_notification=True,
```

Comment:
I think you can skip passing this parameter as we already have a default for it. If we want to make it mandatory to pass it while calling this function, then the code can be updated accordingly, but it is fine to skip it for now and we can do the same in other places as well.

We can instead just pass it only when we are actually testing the behavior for this parameter.